src/styles/index.scss

```diff
@@ -1,3 +1,10 @@
+@import 'header';
+@import 'card';
+@import 'stars';
+
 body {
   margin: 0;
+  font-family: Roboto, serif, sans-serif;
```

Comment:
The `font-family` declaration includes both 'serif' and 'sans-serif', which are typically mutually exclusive. Consider specifying only one as a fallback to 'Roboto' for clarity.